distribution/src/resources/drill-conf

```diff
@@ -1,19 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Not sure if this change is necessary. Please check here and below.

Reply:
It is good for python projects and for FreeBSD and OpenBSD systems.
None of these points satisfy Drill project. Therefore I will revert that changes and change `/usr/bin/env bash` in all other files.

Reply:
Done